packages/cli/dist/commands/generate/index.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
@jimezesinachi please be more careful with your MR: this file shouldn't have been commited

Reply:
The same is true for a bunch of other .js/.map.js files below